src/Backend/Modules/Pages/Resources/config/services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+    fork.pages.copy_module_to_other_locale:
+        class: Backend\Modules\Pages\Command\CopyModuleToOtherLocale
+        public: true
+        tags:
+            - { name: fork.copy_module_to_other_locale }
```

Comment:
It looks like your editor isn't set up to add an EOF character at the end of every file. This is a best practice. 

This should btw be covered by the editorconfig plugin (if you have it installed. PS: you should have it installed 😉 )

Reply:
Fixed